File: Sorting/python/01_수정렬하기.py
# fast campus 강의
# https://www.acmicpc.net/problem/2750
# 1

N = int(input())
sort_list = []
for _ in range(N):
    temp = int(input())
    sort_list.append(temp)

sort_list.sort()

for i in range(N):
    print(sort_list[i])

File: Sorting/c++/01_수정렬하기.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/2750
// 2

#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
	int N;
	cin >> N;

	vector<int> arr(N);
	for (int i = 0; i < N; i++)
		cin >> arr[i];

	sort(arr.begin(), arr.end());

	for (int i = 0; i < N; i++)
		cout << arr[i] << '\n';
}